tyssue/topology/sheet_topology.py

```diff
@@ -114,6 +114,31 @@ def type1_transition(
 
 
 def cell_division(sheet, mother, geom, angle=None):
+    """
+     gcourcou 11/1/2020 periodic boundary consideration:
+     cell_division function does not work if a cell rests on a periodic boundary edge this function.
+     Function checks for perodic boundaries 
+     if there are, it checks if dividing cell rests on an edge of the periodic boundaries
+     if so, it displaces the boundaries by a half a period and moves the target cell in the bulk of the tissue
+     It then performs cell division normally and reverts the periodic boundaries to the original configuration
```

Comment:
If you have time, could you format this as a docstring (look at other docstrings in the code), with a `Parameters` section
and a `Note` section with your remark on PBC handling

Reply:
Implemented it in my best of ability.